wisp/wisp.php

```diff
@@ -597,3 +680,155 @@ function wisp_ClientArea(array $params) {
         // Ignore
     }
 }
+
+/* Utility Functions */
+function getNodes(array $params, int $loc_id) {
+    /*
+        Fetches and returns all nodes with a specific location_id
+    */
+    $filteredNodes = Array();
+    $nodes = wisp_API($params, 'nodes/');
```

Comment:
The `nodes/` should be `nodes` since wispv2 will print an error when using nodes/